src/server.jl

```diff
@@ -105,6 +107,54 @@ function append_slash(url_str::AbstractString)
     return string(endswith(uri.path, "/") ? uri : merge(uri; path = uri.path * "/"))
 end
 
+"""
+    get_dir_list(target::AbstractString) -> index_page::AbstractString
+
+Get dir list of current path. generate a index page html.
+target is current dir path.
```

Comment:
Suggestions:

* replace `target` by `dir` (`get_dir_list(dir::...)` 
* Docstring: ``Generate list of content at path `dir`.``